apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/preadj/R4ClaimResourceProvider.java

```diff
@@ -13,13 +15,17 @@
 @Component
 public class R4ClaimResourceProvider extends AbstractR4ResourceProvider<Claim> {
 
+  private static final Map<String, ResourceTypeV2<Claim>> claimTypeMap =
+      ImmutableMap.of("fiss", ClaimTypeV2.F);
```

Comment:
Will the type parsing accept "fiss" or "FISS" as a query parameter?

Reply:
it will accept both, if you look at AbstractR4ResourceProvider#parseClaimTypes()
```Set<ResourceTypeV2<T>> parseClaimTypes(@Nonnull TokenAndListParam types) {
    return types.getValuesAsQueryTokens().get(0).getValuesAsQueryTokens().stream()
        .map(TokenParam::getValue)
        .map(String::toLowerCase) // <-- Requested types are transformed to lowercase
        .map(getResourceTypeMap()::get) // <-- Then pulled from the implemented map
        .filter(Objects::nonNull)
        .collect(Collectors.toSet());
  }